File: SteamDroid/src/com/flyingpie/steamdroid/api/Protocol.java
package com.flyingpie.steamdroid.api;

public class Protocol {

	public static final String SPLIT_STRING = " ";
	
	public class Client
	{
		public static final String LIST_FRIENDS = "list_friends";
		public static final String CHAT_SEND = "chat_send";
		public static final String SET_STATUS = "set_status";
		public static final String AUTH_SEND = "auth_send";
		public static final String LOGOUT = "logout";
	}
	
	public class Server
	{
		public static final String LIST_FRIENDS = "list_friends";
		public static final String NO_SUCH_FRIEND = "no_such_friend";
		public static final String CHAT_RECEIVED = "chat_received";
		public static final String AUTH_REQUEST = "auth_request";
		public static final String AUTH_SENDING = "auth_sending";
		public static final String LOGGED_IN = "logged_in";
		public static final String LOGGED_OUT = "logged_out";
		public static final String NOT_ALLOWED = "not_allowed";
		public static final String FRIEND_STATE_CHANGED = "friend_state_changed";
		public static final String SUCCESS = "success";
		public static final String INVALID_ARGUMENT = "invalid_argument";
		public static final String UNKNOWN_COMMAND = "unknown_command";
	}
}

File: SteamDroidServer/SteamDroidServer/Client/Protocol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SteamDroidServer.Client
{
    public class Protocol
    {
        public const String SplitString = " ";

        public class Client
        {
            public const String ListFriends = "list_friends";
            public const String ChatSend = "chat_send";
            public const String SetStatus = "set_status";
            public const String AuthSend = "auth_send";
            public const String Logout = "logout";
        }

        public class Server
        {
            public const String ListFriends = "list_friends";
            public const String NoSuchFriend = "no_such_friend";
            public const String ChatReceived = "chat_received";
            public const String AuthRequest = "auth_request";
            public const String AuthSending = "auth_sending";
            public const String LoggedIn = "logged_in";
            public const String LoggedOut = "logged_out";
            public const String FriendStateChanged = "friend_state_changed";
            public const String NotAllowed = "not_allowed";

            public const String Success = "success";
            public const String InvalidArgument = "invalid_argument";
            public const String UnknownCommand = "unknown_command";
        }
    }
}
